src/assets/TokensController.test.ts

```diff
@@ -34,24 +37,38 @@ describe('TokensController', () => {
     });
   });
 
-  it('should add token', async () => {
+  it('should add a token', async () => {
+    const supportsInterfaceStub = sinon.stub().returns(Promise.resolve(false));
+    await sinon
```

Comment:
I don't think `await` is needed here :thinking: The fake function returns a Promise, but it's just being put in place here, not executed.

Reply:
You're right! Not sure why I thought that was necessary!

Reply:
removed throughout the test